crates/frontend/src/pages/experiment_list/utils.rs

```diff
@@ -115,20 +144,58 @@ pub fn experiment_table_columns() -> Vec<Column> {
                 }
                 .into_view()
             },
+            ColumnSortable::No,
         ),
-        Column::new("chosen_variant".to_string(), None, |value: &str, _| {
-            let label = match value {
-                "null" => "¯\\_(ツ)_/¯".to_string(),
-                other => other.to_string(),
-            };
+        Column::new(
+            "chosen_variant".to_string(),
+            None,
+            |value: &str, _| {
+                let label = match value {
+                    "null" => "¯\\_(ツ)_/¯".to_string(),
+                    other => other.to_string(),
+                };
 
-            view! {
-                <span>{label}</span>
-            }
-            .into_view()
-        }),
-        Column::default("created_at".to_string()),
+                view! {
+                    <span>{label}</span>
+                }
+                .into_view()
+            },
+            ColumnSortable::No,
+        ),
+        Column::default_with_sort(
+            "created_at".to_string(),
+            ColumnSortable::Yes {
+                sort_fn: Callback::new(move |_| {
+                    let filters = filters_rs.get();
+                    let sort_by = filters.sort_by.unwrap_or_default().flip();
+                    let new_filters = ExperimentListFilters {
+                        sort_on: Some(ExperimentSortOn::CreatedAt),
+                        sort_by: Some(sort_by),
+                        ..filters
+                    };
+                    filters_ws.set(new_filters);
+                }),
```

Comment:
we dont need to update the filters here
`sort_fn` can simple take a parameter which says what is the current state here
based on that things can be changed and rendered

alternatively, we can also check for `current_sort_on == ExperimentSortOn::CreatedAt` just like its being done for current value

Reply:
Discussed this offline and resolved